cpp/src/phonenumbers/phonenumberutil.cc

```diff
@@ -806,12 +818,9 @@ PhoneNumberUtil::PhoneNumberUtil()
   // Sort all the pairs in ascending order according to country calling code.
   std::sort(country_calling_code_to_region_code_map_->begin(),
             country_calling_code_to_region_code_map_->end(), OrderByFirst());
-}
 
-PhoneNumberUtil::~PhoneNumberUtil() {
-  gtl::STLDeleteContainerPairSecondPointers(
-      country_calling_code_to_region_code_map_->begin(),
-      country_calling_code_to_region_code_map_->end());
+  // Sets has_used_metadata_ to true for GetInstance()
```

Comment:
This comment would be more useful if it mentioned that this prevents the metadata from being read two times in GetInstance.